modules/olm-overview.adoc

```diff
@@ -27,7 +27,7 @@ ifdef::cluster-caps[]
 
 [discrete]
 == Purpose
-endif::[]
```

Comment:
This will never get a bot comment because the line is removed. And the Vale linting happens on a part of the file that isn't modified.